middle_end/flambda2/from_lambda/lambda_to_flambda_primitives.ml

```diff
@@ -1487,11 +1501,15 @@ let convert_lprim ~big_endian (prim : L.primitive) (args : Simple.t list list)
     let arg1 = unbox_bint kind arg1 in
     let arg2 = unbox_bint kind arg2 in
     [ tag_int
-        (Binary (convert_boxed_integer_comparison_prim kind comp, arg1, arg2))
-    ]
+        (Binary
+           ( convert_unboxed_integer_comparison_prim
+               (Primitive.unbox_integer kind)
```

Comment:
I wonder if "Primitive.unboxed_integer" or something might be clearer - "unbox_integer" sounds like an operation on integers themselves.